packages/relay/src/synchornizer.ts

```diff
@@ -59,30 +59,124 @@ export class UnirepSocialSynchronizer extends Synchronizer {
                 transactionHash,
             },
         })
-        if (!findPost) return
 
         const epochKey = BigInt(event.topics[1]).toString(10)
         const postId = BigInt(event.topics[2]).toString()
         const epoch = Number(event.topics[3])
         const content = decodedData.content
 
-        db.upsert('Post', {
-            where: {
-                _id: findPost._id,
-            },
-            create: {
+        console.log('----handlePost----')
+        console.log('Epoch key:', epochKey)
+        console.log('Post id:', postId)
+        console.log('Epoch:', epoch)
+        console.log('Content:', content)
+        console.log('--------------------')
+
+        if (findPost) {
```

Comment:
Is this correct? The original logic is post is not found then skip the processing
Here change to post is not found, then create new one